Source/Core/VideoCommon/AbstractTexture.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include "Common/CommonTypes.h"
+#include "Common/MathUtil.h"
+#include "VideoCommon/TextureConfig.h"
+#include "VideoCommon/VideoCommon.h"
+
```

Comment:
I doubt it's a large change, and the enumeration doesn't make sense being in VideoCommon.h.